container/base-test/Dockerfile

```
@@ -49,7 +49,7 @@ ENV VIRTUAL_ENV="$VIRTUAL_ENV_PARENT/.venv"
 RUN python3 -m venv $VIRTUAL_ENV
 ENV PATH="$VIRTUAL_ENV/bin:$PATH"
 
-COPY Pipfile "$VIRTUAL_ENV_PARENT"
+COPY Pipfile* "$VIRTUAL_ENV_PARENT/"
```

```suggestion
COPY Pipfile Pipfile.lock "$VIRTUAL_ENV_PARENT/"
```
nitpick/style: explicitly control what is included. 
wildcard is also fine for me.